linker/src/lib.rs

```diff
@@ -185,52 +209,67 @@ mod test {
     pub fn compile_simple_sum() {
         let expectation = r#"
 namespace main(1024);
+pol commit XInv;
+pol commit XIsZero;
+XIsZero = (1 - (X * XInv));
+(XIsZero * X) = 0;
+(XIsZero * (1 - XIsZero)) = 0;
+pol commit _sigma;
+_sigma' = ((1 - first_step') * (_sigma + instr_return));
+(_sigma * (_function_id - 10)) = 0;
 pol constant first_step = [1] + [0]*;
 pol commit pc;
 pol commit X;
-(first_step * A) = 0;
 pol commit reg_write_X_A;
 pol commit A;
-(first_step * CNT) = 0;
 pol commit reg_write_X_CNT;
 pol commit CNT;
-pol commit XInv;
-pol commit XIsZero;
-XIsZero = (1 - (X * XInv));
-(XIsZero * X) = 0;
-(XIsZero * (1 - XIsZero)) = 0;
 pol commit instr_jmpz;
 pol commit instr_jmpz_param_l;
 pol commit instr_jmp;
 pol commit instr_jmp_param_l;
 pol commit instr_dec_CNT;
 pol commit instr_assert_zero;
 (instr_assert_zero * (XIsZero - 1)) = 0;
+pol commit instr_return;
+pol commit instr__jump_to_operation;
+pol commit instr__reset;
+pol commit instr__loop;
```

Comment:
why the double underscore

Reply:
Because they are the flags for the instructions `_jump_to_operation` `_reset` `_loop`
This is my way of indicating that they are not user facing.

Reply:
ok